homeassistant/components/roon/strings.json

```diff
@@ -15,9 +17,6 @@
     "error": {
       "invalid_auth": "[%key:common::config_flow::error::invalid_auth%]",
       "unknown": "[%key:common::config_flow::error::unknown%]"
-    },
-    "abort": {
-      "already_configured": "[%key:common::config_flow::abort::already_configured_device%]"
```

Comment:
We currently don't guard against setting up the same host + port combo as an existing config entry when adding a new config entry. We may want to add that. We have this helper:
https://github.com/home-assistant/core/blob/6a92081e831262a431345a42f23181af9ecdd844/homeassistant/config_entries.py#L1216-L1220